website/src/blogs/2019-01-31-introduction-to-openebs-cstor-pools-and-considerations-during-k8s-upgrades.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Introduction to OpenEBS cStor Pools and considerations during K8s upgrades
+author: Uma Mukkara
+author_info: Contributor at openebs.io, Co-founder & COO@MayaData. Uma led product development in the early days of MayaData (CloudByte).
+date: 31-01-2019
+tags: Cloud Native Storage, Kubernetes Upgrades, Persistent Storage, Persistent Volume, Kubernetes
+excerpt: Developers and DevOps administrators like Kubernetes for the way it has eased the tasks in their daily lives. It has changed the way applications are designed, developed and managed. 
+---
+
+> My Kubernetes was just upgraded — what happened to my data?
+
+> And what if I’m using ephemeral disks and thin provisioning to save money?
+
+Developers and DevOps administrators like Kubernetes for the way it has eased the tasks in their daily lives. It has changed the way applications are designed, developed and managed. However, the churn of features and capabilities of Kubernetes continues to be high and frequent upgrades to the platform are not uncommon. Kubernetes development releases are coming out on quarterly basis and production upgrades are common every 4 to 6 months. What is more — if you are running on a Kubernetes service you typically are not 100% in control of when and how those upgrades are performed. The infrastructure software that runs on Kubernetes need to be cognitive of these upgrades so that application developers and Kubernetes administrators do not need to do any hand holding during such upgrades.
```

Comment:
```suggestion
Developers and DevOps administrators like Kubernetes for the way it has eased the tasks in their daily lives. It has changed the way applications are designed, developed, and managed. However, the churn of features and capabilities of Kubernetes continues to be high and frequent upgrades to the platform are not uncommon. Kubernetes development releases are coming out on a quarterly basis and production upgrades are common every 4 to 6 months. What is more — if you are running on a Kubernetes service you typically are not 100% in control of when and how those upgrades are performed. The infrastructure software that runs on Kubernetes needs to be cognitive of these upgrades so that application developers and Kubernetes administrators do not need to do any hand-holding during such upgrades.
```